koku/masu/database/trino_sql/gcp/openshift/gcp_openshift_daily.sql

```diff
@@ -0,0 +1,151 @@
+-- Now create our proper table if it does not exist
+CREATE TABLE IF NOT EXISTS hive.{{schema | sqlsafe}}.gcp_openshift_daily
+(
+    invoice_month varchar,
+    billing_account_id varchar,
+    project_id varchar,
+    usage_start_time timestamp,
+    service_id varchar,
+    sku_id varchar,
+    system_labels varchar,
+    labels varchar,
+    cost_type varchar,
+    location_region varchar,
+    resource_name varchar,
+    project_name varchar,
+    service_description varchar,
+    sku_description varchar,
+    usage_pricing_unit varchar,
+    usage_amount_in_pricing_units double,
+    currency varchar,
+    cost double,
+    daily_credits double,
+    resource_global_name varchar,
+    ocp_matched boolean,
+    matched_tag varchar,
+    gcp_source varchar,
+    ocp_source varchar,
+    year varchar,
+    month varchar,
+    day varchar
```

Comment:
Is there a reason we are renaming `ocp_source_uuid` to `ocp_source` and `source` to `gcp_source`?

Reply:
To match what we have in the other trino tables because things where not happy